Formula/pastebinit.rb

```diff
@@ -20,7 +19,7 @@ def install
     inreplace "pastebinit", "/usr/local/etc/pastebin.d", etc + "pastebin.d"
     system "docbook2man", "pastebinit.xml"
     bin.install "pastebinit"
-    etc.install "pastebin.d"
+    cp_r "pastebin.d", etc
```

Comment:
Maybe we can prompt a message like `The new configurations is updated. Do you want to use the new configurations instead of the old ones? [Y/n]` to user. When they enter `Y`, we replace `pastebin.d` folder. What do you think?